plaso/lib/aul/nonactivity.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+"""The Apple Unified Logging (AUL) Non-activity chunk parser."""
+import base64
+
+from dfdatetime import apfs_time as dfdatetime_apfs_time
+
+from plaso.lib import errors
+
+from plaso.lib.aul import constants
+from plaso.lib.aul import dsc
+from plaso.lib.aul import formatter
+from plaso.parsers import aul
+from plaso.parsers import logger
+
+
+class NonactivityParser():
+  """Non-activity data chunk parser"""
+
+  _NON_ACTIVITY_SENINTEL = 0x80000000
+
+  def ParseNonActivity(self, tracev3, parser_mediator, tracepoint, proc_info,
```

Comment:
This is quite long, is it possible to break this into smaller logical functions?